oak_runtime/src/introspect.rs

```diff
@@ -95,6 +95,42 @@ fn find_ids(path: &str, kind: &str) -> Option<(u64, u64)> {
     Some((id, subid))
 }
 
+// Looks for a matching file generated by webpack and returns it
```

Comment:
I don't think "generated by webpack" is needed in the comment &ndash; wouldn't the code work the same regardless of what tool generates the directory contents?

Reply:
Yes, added for specificity, but happy to rm that part.